docs/02-app/02-api-reference/05-next-config-js/partial-prerendering.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Partial Prerendering
+description: Learn how to enable Partial Prerendering (experimental) in Next.js 14.
+---
+
+Partial Prerendering allows static portions of a route to be prerendered and served from the cache with dynamic holes streamed in, all in a single HTTP request.
+
+You can enable Partial Prerendering by setting the experimental `ppr` flag:
+
+```js filename="next.config.js"
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  experimental: {
+    ppr: true,
+  },
+}
+
+module.exports = nextConfig
+```
+
```

Comment:
Because this is experimental, do we want to add a section on known issues / caveats? Beyond just the note above?

Reply:
Added one caveat around client side navigations